test/preprocessor_benchmark.py

```diff
@@ -0,0 +1,11 @@
+# import Preprocessor
+import time
+
+def preprocessor_benchmark(cfg_path = None):
+  time_start = time.time()
+  time.sleep(1.234)
+  ### calls the actual Preprocessor method
+  # Preprocessor.preprocessor_start(cfg_path)
```

Comment:
Make sure that this gets replaced with a call to the data preprocessor. If your team creates a boilerplate for the preprocessor, you should be able to call those dummy methods.